Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version: 5.7
+// swift-tools-version: 5.8
```

Comment:
I think we need to upgrade to 5.10 as the last pre-6 version to support from now on.

```suggestion
// swift-tools-version: 5.10
```